config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -0,0 +1,606 @@
+    always_run: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        # We pick some version for which there are pre-built images for kind.
+        # Update only when the newer version is known to not cause issues,
+        # otherwise presubmit jobs may start to fail for reasons that are
+        # unrelated to the PR. Testing against the latest Kubernetes is covered
+        # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi#Summary).
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.13.3"
+        - name: CSI_PROW_KUBERNETES_DEPLOYMENT
+          value: "1.13"
+        # use unmodified deployment
+        - name: CSI_PROW_BUILD_JOB
+          value: "false"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2000m
+  - name: pull-sig-storage-csi-driver-host-path-deployment-1-13-on-kubernetes-1-14
+    # Experimental job, explicitly needs to be started with /test.
+    # This can be enabled once the components have the necessary configuration
+    # for this combination of deployment+Kubernetes.
+    always_run: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        # We pick some version for which there are pre-built images for kind.
+        # Update only when the newer version is known to not cause issues,
+        # otherwise presubmit jobs may start to fail for reasons that are
+        # unrelated to the PR. Testing against the latest Kubernetes is covered
+        # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi#Summary).
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.14.0"
+        - name: CSI_PROW_KUBERNETES_DEPLOYMENT
+          value: "1.13"
+        # use unmodified deployment
+        - name: CSI_PROW_BUILD_JOB
+          value: "false"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2000m
+  - name: pull-sig-storage-csi-driver-host-path-1-13-on-kubernetes-master
```

Comment:
You mentioned that we shouldn't be running pull jobs against K8s master?

Reply:
The goal was to have the job in Prow, but never enable it by default (in contrast to the other jobs, which will get enabled by default eventually):
```
    # Experimental job, explicitly needs to be started with /test.
    # This cannot be enabled by default because there's always the risk
    # that something changes in master which breaks the pre-merge check.
```

Then if we see a PR which is worthwhile testing on master right away, we can explicitly request a job run with `/test`.

This is not quite the same as testing against a version of Kubernetes revision specified inside a PR (see my other comment), because here we always use the latest Kubernetes without updating the E2E tests.